db/schema.rb

```diff
@@ -405,8 +405,8 @@
   create_table "provider_users_providers", force: :cascade do |t|
     t.bigint "provider_id", null: false
     t.bigint "provider_user_id", null: false
-    t.datetime "created_at", default: -> { "now()" }, null: false
-    t.datetime "updated_at", default: -> { "now()" }, null: false
+    t.datetime "created_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "updated_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
Does this fluctuate because of version differences? Might be worth leaving this hunk out until we resolve this and standardise across our installs somehow.

Reply:
Yes, I think Postgres 9 uses `now`, later versions its `CURRENT_TIMESTAMP`. Will remove that hunk.